lib/brick_seeker_web/live/live_helpers.ex

```diff
@@ -57,4 +57,13 @@ defmodule BrickSeekerWeb.LiveHelpers do
     |> JS.hide(to: "#modal", transition: "fade-out")
     |> JS.hide(to: "#modal-content", transition: "fade-out-scale")
   end
+
+  def allow_ecto_sandbox(socket) do
```

Comment:
This is really interesting. It looks like a side effect function that adds the liveview socket to the ecto sandbox? Does this provide a workaround to https://github.com/phoenixframework/phoenix_live_view/issues/708#issuecomment-629665050?

How does this function behave in a production environment?

Reply:
https://hexdocs.pm/phoenix_ecto/Phoenix.Ecto.SQL.Sandbox.html#module-liveview
that was copied from the docs. the sandbox plug is only enabled in the testing environment so there shouldn't be any effect in production

Reply:
I would be more comfortable if there was a branch condition earlier that skipped this function's execution entirely if the environment isn't sandboxed, but I think that can be discussed and addressed in future commits.